--- reiser4progs-0.4.20/configure.in.orig	2003-12-23 10:03:27.000000000 +0100
+++ reiser4progs-0.4.20/configure.in	2003-12-28 13:52:26.714312424 +0100
@@ -41,80 +41,6 @@
 AC_DEFINE_UNQUOTED(LIBREISER4_MAX_INTERFACE_VERSION, $LIBREISER4_BINARY_AGE)
 AC_DEFINE_UNQUOTED(LIBREISER4_MIN_INTERFACE_VERSION, $LIBREISER4_INTERFACE_AGE)
 
-# Macro for checking libaal
-AC_DEFUN([AC_CHECK_LIBAAL],
-[
-
-AC_CHECK_LIB($1, aal_device_open, ,
-	[AC_MSG_ERROR([lib$1 not found; install libaal available at \
-http://www.namesys.com/pub/libaal])]
-$4)
-
-# Check for headers and library
-AC_CHECK_HEADER(aal/aal.h, ,
-    	[AC_MSG_ERROR([<aal/aal.h> not found])] 
-$4)
-
-AC_MSG_CHECKING(for lib$1 version = $2)
-
-AC_TRY_LINK_FUNC(libaal_version,,
-    	AC_MSG_RESULT(failed)
-    	AC_MSG_ERROR([lib$1 can't execute test]))
-
-# Get major, minor, and micro version from arg MINIMUM-VERSION
-libaal_config_major_version=`echo $2 | \
-    sed 's/\([[0-9]]*\).\([[0-9]]*\).\([[0-9]]*\)/\1/'`
-libaal_config_minor_version=`echo $2 | \
-    sed 's/\([[0-9]]*\).\([[0-9]]*\).\([[0-9]]*\)/\2/'`
-libaal_config_micro_version=`echo $2 | \
-    sed 's/\([[0-9]]*\).\([[0-9]]*\).\([[0-9]]*\)/\3/'`
-
-# Compare MINIMUM-VERSION with libaal version
-AC_TRY_RUN([
-#include <stdio.h>
-#include <stdlib.h>
-#include <aal/aal.h>
-
-int main() {
-    	const char *version;    
-    	int major, minor, micro;
-	
-    	if (!(version = libaal_version()))
-		exit(1);
-		
-    	if (sscanf(version, "%d.%d.%d", &major, &minor, &micro) != 3) {
-		printf("%s, bad version string\n", version);
-		exit(1);
-    	}
-	
-    	if ((major >= $libaal_config_major_version) &&
-		((major == $libaal_config_major_version) && 
-		(minor >= $libaal_config_minor_version)) &&
-		((major == $libaal_config_major_version) && 
-		(minor == $libaal_config_minor_version) && 
-		(micro >= $libaal_config_micro_version))) 
-    	{
-		return 0;
-    	}
-
-    	printf("\nAn old version of libaal (%s) was found.\n",
-		version);
-
-    	printf("You need a version of libaal newer than %d.%d.%d.\n", 
-		$libaal_config_major_version, $libaal_config_minor_version, 
-		$libaal_config_micro_version);
-
-    	printf("You can get it at ftp://ftp.namesys.com/pub/libaal\n");
-    	return 1;
-}
-], 
-    	AC_MSG_RESULT(yes),
-    	AC_MSG_RESULT(no) ; $4,
-    	[echo $ac_n "cross compiling; assumed OK... $ac_c"])
-
-# Execute ACTION-IF-FOUND
-$3])
-
 AC_ARG_WITH(libaal,
 	[  --with-libaal            prefix of where libaal was installed], ,
 )
@@ -466,7 +392,7 @@
 ])
 
 OLD_LIBS="$LIBS"
-AC_CHECK_LIBAAL(aal, 0.4.15, ,exit)
+AC_CHECK_LIBAAL(0.4.15, [LIBS="-laal"],exit)
 AAL_LIBS="$LIBS"
 LIBS="$OLD_LIBS"
 
@@ -474,7 +400,7 @@
 
 if test x$enable_stand_alone = xyes; then
     	OLD_LIBS="$LIBS"
-    	AC_CHECK_LIBAAL(aal-alone, 0.4.15, ,exit)
+    	AC_CHECK_LIBAAL(0.4.15, [LIBS="-laal"],exit)
     	AAL_ALONE_LIBS="$LIBS"
     	LIBS="$OLD_LIBS"
 
