--- reiser4progs-1.0.7/libmisc/ui.c.formatsec	2013-02-14 22:14:31.296635365 +0100
+++ reiser4progs-1.0.7/libmisc/ui.c	2013-02-14 22:14:55.013173548 +0100
@@ -116,7 +116,7 @@ void misc_print_wrap(void *stream, char
 				line_width = 0;
 			}
 
-			fprintf(stream, word);
+			fputs(word, stream);
 
 			line_width += aal_strlen(word);
 
--- reiser4progs-1.0.7/librepair/add_missing.c.formatsec	2013-02-14 22:09:50.018159182 +0100
+++ reiser4progs-1.0.7/librepair/add_missing.c	2013-02-14 22:10:01.418097423 +0100
@@ -65,7 +65,7 @@ static void repair_add_missing_update(re
 	time_str = ctime(&am->stat.time);
 	time_str[aal_strlen(time_str) - 1] = '\0';
 	aal_stream_format(&stream, time_str);
-	aal_mess(stream.entity);
+	aal_mess("%s", stream.entity);
 	aal_stream_fini(&stream);
 }
 
--- reiser4progs-1.0.7/librepair/cleanup.c.formatsec	2013-02-14 22:10:13.598031438 +0100
+++ reiser4progs-1.0.7/librepair/cleanup.c	2013-02-14 22:10:24.067974718 +0100
@@ -26,7 +26,7 @@ static void repair_cleanup_update(repair
 	time_str = ctime(&cleanup->stat.time);
 	time_str[aal_strlen(time_str) - 1] = '\0';
 	aal_stream_format(&stream, time_str);
-	aal_mess(stream.entity);
+	aal_mess("%s", stream.entity);
 	aal_stream_fini(&stream);
 }
 
--- reiser4progs-1.0.7/librepair/disk_scan.c.formatsec	2013-02-14 22:04:53.776430727 +0100
+++ reiser4progs-1.0.7/librepair/disk_scan.c	2013-02-14 22:07:46.108830454 +0100
@@ -45,7 +45,7 @@ static void repair_disk_scan_update(repa
 	time_str = ctime(&ds->stat.time);
 	time_str[aal_strlen(time_str) - 1] = '\0';
 	aal_stream_format(&stream, time_str);
-	aal_mess(stream.entity);
+	aal_mess("%s", stream.entity);
 	aal_stream_fini(&stream);
 }
 
--- reiser4progs-1.0.7/librepair/filter.c.formatsec	2013-02-14 22:08:13.782013871 +0100
+++ reiser4progs-1.0.7/librepair/filter.c	2013-02-14 22:08:34.618567656 +0100
@@ -591,7 +591,7 @@ static void repair_filter_update(repair_
 	time_str = ctime(&fd->stat.time);
 	time_str[aal_strlen(time_str) - 1] = '\0';
 	aal_stream_format(&stream, time_str);
-	aal_mess(stream.entity);
+	aal_mess("%s", stream.entity);
 	aal_stream_fini(&stream);
 }
 
--- reiser4progs-1.0.7/librepair/semantic.c.formatsec	2013-02-14 22:10:38.364563933 +0100
+++ reiser4progs-1.0.7/librepair/semantic.c	2013-02-14 22:10:54.754475141 +0100
@@ -791,7 +791,7 @@ static void repair_semantic_update(repai
 	time_str = ctime(&sem->stat.time);
 	time_str[aal_strlen(time_str) - 1] = '\0';
 	aal_stream_format(&stream, time_str);
-	aal_mess(stream.entity);
+	aal_mess("%s", stream.entity);
 	aal_stream_fini(&stream);
 }
 
--- reiser4progs-1.0.7/librepair/twig_scan.c.formatsec	2013-02-14 22:09:27.758279774 +0100
+++ reiser4progs-1.0.7/librepair/twig_scan.c	2013-02-14 22:09:41.601538112 +0100
@@ -119,7 +119,7 @@ static void repair_twig_scan_update(repa
 	time_str[aal_strlen(time_str) - 1] = '\0';
 	
 	aal_stream_format(&stream, time_str);
-	aal_mess(stream.entity);
+	aal_mess("%s", stream.entity);
 	aal_stream_fini(&stream);
 }
 
--- reiser4progs-1.0.7/progs/debugfs/browse.c.formatsec	2013-02-14 22:17:37.565626259 +0100
+++ reiser4progs-1.0.7/progs/debugfs/browse.c	2013-02-14 22:18:01.202164876 +0100
@@ -27,7 +27,7 @@ static errno_t debugfs_reg_cat(reiser4_o
 		if (read <= 0)
 			break;
 
-		printf(buff);
+		fputs(buff, stdout);
 	}
 
 	return 0;
--- reiser4progs-1.0.7/progs/debugfs/print.c.formatsec	2013-02-14 22:18:13.672097311 +0100
+++ reiser4progs-1.0.7/progs/debugfs/print.c	2013-02-14 22:18:27.042024889 +0100
@@ -24,7 +24,7 @@ errno_t debugfs_print_stream(aal_stream_
 		if ((size = aal_stream_read(stream, buff, size)) <= 0)
 			return size;
 		
-		printf(buff);
+		fputs(buff, stdout);
 	}
 
 	return 0;
