diff -burN reiser4progs-0.5.6.orig/configure.in reiser4progs-0.5.6/configure.in
--- reiser4progs-0.5.6.orig/configure.in	2004-08-08 14:38:42.533258288 +0200
+++ reiser4progs-0.5.6/configure.in	2004-08-08 14:46:28.951351960 +0200
@@ -577,13 +577,7 @@
 
 CFLAGS="$CFLAGS -W -Wall"
 
-if test x$enable_debug = xyes; then
-    	GENERIC_CFLAGS="$GENERIC_CFLAGS -O1 -g"
-    	ALONE_CFLAGS="$ALONE_CFLAGS -O1 -g"
-else
-    	GENERIC_CFLAGS="$GENERIC_CFLAGS -O3"
-    	ALONE_CFLAGS="$ALONE_CFLAGS -Os"
-
+if test x$enable_debug != xyes; then
     	if test x$uninitialized = xyes; then
        		CFLAGS="$CFLAGS -Wuninitialized"
     	fi
