diff -burN reiser4progs-0.4.12-orig/progs/mkfs/Makefile.am reiser4progs-0.4.12/progs/mkfs/Makefile.am
--- reiser4progs-0.4.12-orig/progs/mkfs/Makefile.am	2003-09-02 14:18:26.336744008 +0200
+++ reiser4progs-0.4.12/progs/mkfs/Makefile.am	2003-09-02 14:36:50.599870496 +0200
@@ -10,13 +10,13 @@
 INCLUDES		= -I$(top_srcdir)/include
 
 uninstall-make_reiser4:
-	if test -f $(sbindir)/make_reiser4; then \
-	    rm -f $(sbindir)/make_reiser4; \
+	if test -f $(DESTDIR)$(sbindir)/make_reiser4; then \
+	    rm -f $(DESTDIR)$(sbindir)/make_reiser4; \
 	fi;
 
 install-make_reiser4: mkfs.reiser4 uninstall-make_reiser4
-	if test -f $(sbindir)/mkfs.reiser4; then \
-	    ln $(sbindir)/mkfs.reiser4 $(sbindir)/make_reiser4; \
+	if test -f $(DESTDIR)$(sbindir)/mkfs.reiser4; then \
+	    ln $(DESTDIR)$(sbindir)/mkfs.reiser4 $(DESTDIR)$(sbindir)/make_reiser4; \
 	fi;
 
 install-data-local: install-make_reiser4
