diff -burN reiser4progs-0.4.11.orig/configure.in reiser4progs-0.4.11/configure.in
--- reiser4progs-0.4.11.orig/configure.in	2003-08-13 21:01:49.292634280 +0200
+++ reiser4progs-0.4.11/configure.in	2003-08-13 21:02:44.233282032 +0200
@@ -420,14 +420,6 @@
 -DENABLE_SYMLINKS -DENABLE_R5_HASH -DENABLE_FNV1_HASH -DENABLE_RUPASOV_HASH \
 -DENABLE_TEA_HASH -DENABLE_LARGE_KEYS -DENABLE_SHORT_KEYS"
 
-if test x$enable_debug = xyes; then
-    GENERIC_CFLAGS="$GENERIC_CFLAGS -O0 -g"
-    ALONE_CFLAGS="$ALONE_CFLAGS -O0 -g"
-else
-    GENERIC_CFLAGS="$GENERIC_CFLAGS -O2"
-    ALONE_CFLAGS="$ALONE_CFLAGS -Os"
-fi
-
 CFLAGS="$CFLAGS -W -Wall -Wno-unused"
 if test x$enable_Werror = xyes; then
     CFLAGS="$CFLAGS -Werror"
